go.mod

```diff
@@ -4,8 +4,11 @@ go 1.22.0
 
 toolchain go1.22.4
 
+replace github.com/edgelesssys/contrast/node-installer => ./node-installer
```

Comment:
As previously discussed in-person, I don't think using a replace here makes any sense. If you need that code as part of the main module, just make it part of the main module.

Reply:
See the rationale behind this in https://github.com/edgelesssys/contrast/commit/249d627b443ba28cd81a66a4ff66243b4a108006

I think this is the best we can currently get to not have to rebuild the node-installer each time contrast changes.